tyr-runner/src/main/java/org/jboss/tyr/model/json/SnapshotDependencies.java

```diff
@@ -0,0 +1,18 @@
+package org.jboss.tyr.model.json;
+
+import javax.json.bind.annotation.JsonbProperty;
+import java.math.BigDecimal;
+import java.util.List;
+
+public class SnapshotDependencies {
+
+    @JsonbProperty("count")
+    public BigDecimal snapshotDependencyCount;
+
+    @JsonbProperty("snapshot-dependency")
+    public List<SnapshotDependency> snapshotDependencies;
+
+    public SnapshotDependencies() {
+
+    }
```

Comment:
also, why is the default constructor not enough? Also in other Json classes. Did you try to run the app without these empty constructors?

Reply:
there you are right :) these I don't need because of default constructor...sorry for that, they are only remains when I was trying to use parametric constructor too..removing them :)